dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/sequence/SequenceReader.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.dbsnpimporter.sequence;
+
+import java.util.NoSuchElementException;
+
+/**
+ * Interface for readers than returns nucleotide subsequences of a given sequence source (like a fasta file or a WS)
+ */
+public interface SequenceReader {
```

Comment:
I was keeping it just in case the cache annotations could be just applied over interfaces (I think they can be applied over classes as well but with a more complex configuration). Like it seems we are not going to use the cache yet, I will delete this interface